BETA.md

```diff
@@ -1,69 +1,96 @@
 
-- `cd vscode-extension`
-- `npm install`
-- `npm run build`
-- `code --install-extension vscode-metals-0.0.1.vsix`
+The VSCode extension is not yet published on the Marketplace, so you'll need to
+build it locally. For this step, `node`, `npm` and VS Code are required.
+
+* `cd vscode-extension`
+* `npm install`
+* `npm run build`
+* `code --install-extension vscode-metals-0.0.1.vsix`
+
+Make sure you have the
+["Scala Syntax"](https://marketplace.visualstudio.com/items?itemName=daltonjorge.scala)
+plugin installed in VS Code.
 
 ## Step 3 - publish the server locally
+
 From the repo root run `sbt publishLocal`
 
 # Per-project setup
+
 These steps are required on each project.
 
 ## Step 1 - add semanticdb-scalac compiler plugin to your project
-Some features of the server rely on the artifacts produced by the [`semanticdb-scalac`](http://scalameta.org/tutorial/#semanticdb-scalac) compiler plugin.
-There are two alternative ways to install `semanticdb-scalac`.
 
-The first option is to enable `semanticdb-scalac` permanently for your project in `build.sbt` with:
+Some features like definition/references/hover rely on artifacts produced by a
+compiler plugin called `semanticdb-scalac`. There are two alternative ways to
+install `semanticdb-scalac`.
+
+The first option is to enable `semanticdb-scalac` permanently for your project
+in `build.sbt` with:
 
 ```scala
-libraryDependencies += compilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
+ addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
 scalacOptions += "-Yrangepos"
 ```
 
-The second option is to enable `semanticdb-scalac` only for an active sbt session by running `semanticdbEnable` from the sbt shell.
+The second option is to enable `semanticdb-scalac` only for an active sbt
+session by running `semanticdbEnable` from the sbt shell.
+
 ```scala
 $ sbt
 > semanticdbEnable // automatically runs libraryDependencies += compilerPlugin(...)
 > compile // re-compile project with semanticdb-scalac compiler plugin
 > ...
 ```
-As soon as you exit the sbt shell you need to re-run `semanticdbEnable` next time you open sbt.
+
+As soon as you exit the sbt shell you need to re-run `semanticdbEnable` next
+time you open sbt.
 
 ## Step 2 - produce the build metadata
+
 In your project of choice, open `sbt` and run `*:metalsEnableCompletions`.
-As mentioned above, this will produce the necessary metadata for the server.
+Running the task produces the necessary metadata for the server to support
+features like completions and goto definition in dependency sources.
 
-> **NOTE**: you will need to repeat this step every time you add a new source file to the project or if you run `sbt clean`
+> **NOTE**: you will need to repeat this step every time you add a new
+> dependency in your build or when you run `sbt clean`. Also,
+> the "`metalsEnableCompletions`" name is misleading since it does not
+> automatically enable completions.
```

Comment:
I think it's better if you rename it in this PR (to whatever is less misleading) than add this note.

Reply:
I agree, done. Renamed to `metalsSetup`.